addons/base_automation/__manifest__.py

```diff
@@ -15,10 +15,11 @@
 Sales Team, or an opportunity which still has status pending after 14 days might
 trigger an automatic reminder email.
     """,
-    'depends': ['base', 'resource', 'mail', 'sms'],
+    'depends': ['base', 'digest', 'resource', 'mail', 'sms'],
```

Comment:
Instead of adding a dependency here, you could buid a module `digest_automation` which is auto-installable.

Reply:
Would suggest to not add a bridge module just for that, as I heard we try to keep module number under control ^^ (each one slowing down registry, ...)

Reply:
If you don't want a bridge module, why not add `base_automation` as a dependency to `digest` and move the code there?

The `base_automation` module, which is managed by the framework team, *should not* depend on modules by other teams. 
BTW: In the past at a customer, we had disgest deinstalled, but we were using base_automation.

Reply:
If we add `base_automation` as a dependency to `digest`, as soon as we install a module that depends on `digest` then `base_automation` will automatically be installed. 
Will that be an issue ? :thinking:

Reply:
Well digest is considered as a base feature. You can deactivate it by removing digests. It is easier than having lot of bridge modules. I think digest is a technical module that could even be in mail, base automation is a functional module (for technical people), so it makes sense to have digest in dependencies of base automation, not the reverse move.

Reply:
``The base_automation module, which is managed by the framework team, should not depend on modules by other teams.``

You mean like mail (us), sms (us) and resources (vidange) :D ?

Reply:
I was only considering mail as a base feature, not sms or digest. Anyways, it's ok like you suggest because we already have these other modules here.

Reply:
In a perfect world I would make bridge modules, but those have a cost and we are told to try to avoid them as possible, hence adding "fake" dependencies ... what matter is that you can de-activate the digest feature by simply killing digest records.